src/loki/worker.cc

```diff
@@ -107,6 +130,7 @@ void loki_worker_t::parse_costing(Api& api, bool allow_none) {
             avoid->set_id(edge.id);
             avoid->set_percent_along(edge.percent_along);
 
+            // TODO: check if still necessary here since shortcuts are in bins now!
```

Comment:
didn't give that a whole lot of thought and to some degree I refused to understand the logic in loki::Search ;) I'm sure you could judge that easier

Reply:
`loki::search` uses regular costing allowed function which by default disallowes shortcuts. i think this is still good because we actually want to find both the regular edges and the shortcuts and `search` wont do that unless you provide a radius and change the allowed function. so yeah no change here for now but somethjign we could think about encapulating rather than having this detail outside of the fucntion